### PR TITLE
Run all fixtures through Prettier

### PR DESCRIPTION
**Pull request by aweary**
We run the test suite through prettier, so it makes sense to do the same with the fixtures.

### CODE REVIEW

File: scripts/prettier/index.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const defaultOptions = {
 };
 const config = {
   default: {
-    patterns: ['src/**/*.js'],
+    patterns: ['src/**/*.js', 'fixtures/**/*.js'],
```
Comment (aweary) [inline scripts/prettier/index.js]: This is the actual config change

Comment (nhunzaker) [inline scripts/prettier/index.js]: Wow. Nice. 👍

**Comment by aweary** [comment]
@nhunzaker I think it should behave the same way our prettier validation works for the `src` folder, meaning it will fail on CI if the formatting is off. Maybe @gaearon can confirm.

**Comment by gaearon** [comment]
Yep.

**Pull request merged**